srcpkgs/feh/template

```diff
@@ -10,9 +10,9 @@ short_desc="Fast and light imlib2-based image viewer"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="MIT-feh"
 homepage="https://feh.finalrewind.org/"
-#changelog="https://raw.githubusercontent.com/derf/feh/master/ChangeLog"
+changelog="https://raw.githubusercontent.com/derf/feh/master/ChangeLog"
 distfiles="${homepage}/feh-${version}.tar.bz2"
-checksum=3ff3e64db0811e2ff14add77d1aab4ae6d16616f5f78fe54649b49f5aa7f4a21
+checksum=074f8527a17fc5add70018a7f3887d78d5bdf545611636b88641f27e9e844795
 
 do_build() {
 	make ${makejobs} CC=$CC PREFIX=/usr exif=1 help=1 stat64=1
```

Comment:
why not `build_style=gnu-makefile` ?

Reply:
For the record, the first template for `feh` version 2.7:
https://github.com/void-linux/void-packages/commit/6c80eedef50ee8d1625697c4e56734f0506991b2

Changed in version 2.13, without comment:
https://github.com/void-linux/void-packages/commit/2f8cfd2cf604071d92da1492fc0b9f8e3660ede4

Reply:
I propose https://github.com/void-linux/void-packages/pull/484